templates/example.html

```diff
@@ -13,9 +13,12 @@
   <body>
     <div class="body">
       {{> header.html}}
-      {{#sections}}
+      {{#data.sections}}
```

Comment:
In order to pass:
        "experimental": function() {
          return (idx++ == 0 && document.metadata.experiment);
Experimental needs to be inside the first section.